core/src/main/java/org/apache/iceberg/rest/HTTPClient.java

```diff
@@ -71,6 +74,12 @@ public class HTTPClient implements RESTClient {
   @VisibleForTesting
   static final String CLIENT_GIT_COMMIT_SHORT_HEADER = "X-Client-Git-Commit-Short";
 
+  private static final String RETRY_STRATEGY = "rest.client.retry-strategy";
+  private static final String REST_MAX_RETRIES = "rest.client.max-retries";
+  private static final String REST_RETRY_INTERVAL_MILLIS = "rest.client.retry-interval-millis";
```

Comment:
Do we need `client` in all these properties? Is it clear what `client` refers to and that it is distinct from `rest`?

Reply:
I'm ok either way, I just figured that having `client` in there makes it slightly more clear that we're targeting the REST HTTP client. Alternatively we could use `rest.http.max-retries`.

Reply:
I've updated this to `rest.http.max-retries`